jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskSummaryDataSetMapper.java

```diff
@@ -29,62 +29,126 @@
     @Override
     public TaskSummary apply(final DataSet dataSet,
```

Comment:
I guess in this case is best to split this in two specific mapper classes, one for each data set.